index.css

```diff
@@ -0,0 +1,64 @@
+
+
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+}
+h1{
+    font-family: "Inter";
+    font-weight: 400;
+    margin: 1rem 0px 0px 1rem;
+}
+.todos {
+    margin: 1rem 0px 0px 1rem;
+}
+
+.todoList {
```

Comment:
Typically CSS classes are named in kebab-case, i.e `todo-list`. Also, don't use plurals in the names. 

-1